core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
+              send_email_for_new_device_enabled        = true
+              send_email_for_password_changed_enabled  = true
+            }
+            ```
+
+  - uid: mondoo-okta-security-factor-enrollment-notifications
+    title: Enable factor enrollment notifications for end users 
+    impact: 80
+    docs: 
+      desc: |
+        Factor enrollment notifications for end users notifies end users of any activity on their account related to multifactor authentication (MFA) factor enrollment.
+
+        ### Okta Recommends 
 
-        Run this query:
+        Enable this email notification so end users are informed about factor enrollment.
 
-          ```bash
-          cnspec run okta -c "okta.policies.signOn.where(_.rules.all(status == "ACTIVE")).all(_.rules.all(actions["signon"]["session"]["usePersistentCookie"] == false))" --organization <YOUR-ORG> --token <OKTA_TOKEN>
-          ```
+        ### Security impact 
 
-        __Use the cnspec shell__
+        High
 
-        1. Launch the cnspec shell:
+        ### User impact 
 
-          ```bash
-          cnspec shell okta ---organization <YOUR-ORG> --token <OKTA_TOKEN>
-          ```
+        Low
 
-        2. Run this query:
+        End users are sent a confirmation email if they or an admin enroll in a new factor for their account.
 
-          ```mql
-          okta.policies.signOn.where(_.rules.all(status == "ACTIVE")).all(_.rules.all(actions["signon"]["session"]["usePersistentCookie"] == false))
-          ```
-      remediation: "### Use the Okta Admin Console\n\n1. Log into the organization as an administrator.\n2. Select **Security > Global Session Policy**. \n3. Edit the rules related to the policy (as well as default policy): under **Persist session cookies across browser sessions**, select **Disabled**.\n6. When you're finished, update the rule. \n"
-  - uid: mondoo-okta-security-okta-limit-superadmin
-    title: Limit the organization to three super administrators
+    variants: 
+      - uid: mondoo-okta-security-factor-enrollment-notifications-runtime
+      - uid: mondoo-okta-security-factor-enrollment-notifications-terraform-hcl
+      - uid: mondoo-okta-security-factor-enrollment-notifications-terraform-plan
+  - uid: mondoo-okta-security-factor-enrollment-notifications-runtime
+    title: Enable factor enrollment notifications for end users 
     impact: 80
-    mql: |
-      okta.users.where(roles.all(type == "SUPER_ADMIN")).length <= 3
+    filters: asset.platform == "okta-org"
+    mql: okta.organization.securityNotificationEmails['sendEmailForFactorEnrollmentEnabled'] == true
     docs:
+      remediation: 
+        - id: console 
+          desc: |
+            ### Configure factor enrollment notifications for end users
+            1. In the Admin Console, go to **Security > General**.
+            2. Under **Security Notification Emails**, click **Edit**.
+            3. Set **MFA enrolled notification email** to **Enabled**.
+            4. Click **Save**.
```

Comment:
```suggestion
            4. Select **Save**.
```

Reply:
@tas50 these are not our steps. they are from Okta's docs. They are copy/pasted.